README.md

```diff
@@ -71,43 +71,49 @@ In either case, you will want to have [Serverless] installed, eg. `npm install -
 
   provider:
     name: aws
-    runtime: haskell
+    runtime: provided
 
   functions:
     myfunc:
-      handler: mypackage.myfunc
-      # Here, mypackage is the Haskell package name and myfunc is the executable
-      # name as defined in the Cabal file. The handler field may be prefixed
-      # with a path of the form `dir1/.../dirn`, relative to `serverless.yml`,
-      # which points to the location where the Haskell package `mypackage` is
-      # defined. This prefix is not needed when the Stack project is defined at
-      # the same level as `serverless.yml`.
+      handler: src/Lib.handler
+      layers:
+        - arn:aws:lambda:us-east-1:785355572843:layer:haskell-runtime:4
```

Comment:
The region should not be hardcoded.

Reply:
Definitely, it should be parametrized, any ideas?

Reply:
Serverless will have the region somewhere, poke around in the `service` or `this.serverless` object. As to how to add it, again, see https://github.com/seek-oss/serverless-haskell/blob/master/serverless-plugin/index.js#L279 as a starting point.